src/main/java/net/minecraftforge/common/Tags.java

```diff
@@ -177,7 +184,23 @@ private static void init(){}
 
```

Comment:
`needs_x_tool` should all be left in `forge`/`neoforge` as they control behavior in forge, they are not for commonality betwen mods.